### DISCUSSION TITLE
How to select distinct value from a collection (GraphQL)?

### DISCUSSION THREAD

**Discussion by joelamltc** [directus/directus#15217] [Archive]
I have a Post data model setup like the below:
- id (PK)
- title
- content
- category_id (FK)
- ...

I want to get the distinct value of category_id from the Post data model. For example, we have some rows of data like the below:
| id | title | content | category_id |
|---|-----|---------|-------------|
| 1 | lorem | lorem | 1 |
| 2 | lorem | lorem | 2 |
| 3 | lorem | lorem | 3 |
| 4 | lorem | lorem | 1 |
| 5 | lorem | lorem | 1 |

Expected:
```json
[{ "category_id": 1 }, { "category_id": 2 }, { "category_id": 3 }]
```

I wonder if Directus currently supported it. Thanks in advance!

**Comment by rijkvanzanten** [comment]
There's currently no way to retrieve the distinct values from a collection. Make sure to upvote https://github.com/directus/directus/discussions/2989 🙂
